Extras/Scripts/sign-bundled.sh

```diff
@@ -13,7 +13,7 @@
 ## (C) 2013 Sam Deane, Elegant Chaos.
 ## Feel free to use and abuse this script - I'd love to hear of improvements.
 
-SHOW_VERBOSE=false
+SHOW_VERBOSE=true
```

Comment:
Definitely want to turn this on after merging?

Reply:
Mike I was going to leave this on because the script is going to be removed soon anyway. I just wanted to have its output there in order to make sure any desired behaviour is not lost.